Makefile

```diff
@@ -17,51 +17,61 @@
 # DOCKER_BUILD=docker build --no-cache
 DOCKER_BUILD=docker buildx build --push --platform linux/amd64,linux/arm64 --output type=registry
 DOCKER_BUILD=docker buildx build --push --platform linux/arm64 --output type=registry
+ARCH=$(shell uname -m)
```

Comment:
This reflects only the host OS, right? In other words, if I call `make` on a `x86_64` operating system, then `ARCH` will contain `x86_64`.

Reply:
I think we could leave the `---platform linux/xxxx` out. By default single arch images are built. 

However, this does not work. I also got 

```

951.3 ld.mold: ./elf/thunks.cc:257: void mold::elf::OutputSection<E>::create_range_extension_thunks(mold::elf::Context<E>&) [with E = mold::elf::ARM64]: Assertion `thunk->size() < max_thunk_size' failed.

``` 
on my arm machine (the same machine used to build the dependencies image, so it should not be an issue of different architectures).

Reply:
> By default single arch images are built.

So you build the images on two different architectures. Then later copy one container over and merge them together into one. I think I understand now.

Reply:
I actually build them using buildx (pushing them right away on docker with the right tag), and then ask buildx to join them (remotely). The merge step does not copy them locally. I'm now trying with the CXX flag to see if this solves the issue on my arm machine.